build/macos-x64/install.dependencies.sh

```diff
@@ -50,16 +50,16 @@ brew uninstall --force little-cms2
 brew uninstall --force libtiff
 brew uninstall --force jasper
 brew uninstall --force libjpeg
-brew uninstall --force jpeg
+brew uninstall --force jpeg-turbo
```

Comment:
Should we not uninstall both of them?

Reply:
I inspected the `brew list --formula` output at the beginning of this file and saw that the `jpeg` package wasn't actually installed, which is why I changed this instead of adding a new line. Happy to put both of them in for future-proofing if you want

Reply:
Aah okay, they probably changed this. We can keep it as is.